README.md

```diff
@@ -1,12 +1,70 @@
+## Which folders are important?
```

Comment:
This readme.md file is no longer accurate and cannot be followed.  Please cleanup or remove and indicate the task that has been created to fix this.